app/avo/actions/restore_rubygem_version.rb

```diff
@@ -0,0 +1,32 @@
+class RestoreRubygemVersion < BaseAction
+  field :version_with_platform, as: :select,
+    options: lambda { |model:, resource:, view:, field:| # rubocop:disable Lint/UnusedBlockArgument
+      Deletion.where(rubygem: model.name).map do |deletion|
+        ["version:#{deletion.number} - platform:#{deletion.platform}", deletion.id]
```

Comment:
maybe use `deletion.version.slug` as the string here?

Reply:
actually -- this might be simpler if we put the action on the Version resource, and only show it when `version.index == false` ?

Reply:
I have pushed a commit here https://github.com/rubygems/rubygems.org/pull/3920/commits/6899f1f98e1087f0d6e1798ed19855802928bfb3